pkg/eventprocessor/impl/eventprocessor_replayhistory_test.go

```diff
@@ -35,14 +35,14 @@ func TestReplayProductionHistory(t *testing.T) {
 	}
 
 	expectedStateHashes := map[tableland.ChainID]string{
```

Comment:
This version of the parser breaks the final hash state. We gotta update the hashes. Not a big deal.

Reply:
out of curiosity, what are these hashed?

Reply:
This is a test where we replay some historical events and calculate the final state of the database. Each chain has a final state hash. This test allows seeing if the change we made broke history and also allows us to see if our execution is deterministic. 

The new parser version breaks history because the statements are smaller (without unneeded spaces). Technically, that does not change the execution of events and user data should be the same. Still, some error messages that we store in receipts tables are affected by the size of the statement, thus, breaking history. Not a big deal though.